### PR TITLE
feat: adding vertical dots [STRY-301]

### PR DESCRIPTION
**Pull request by val-l-hosler**
## What I did

[STRY-301](https://arcpublishing.atlassian.net/browse/STRY-301?atlOrigin=eyJpIjoiOGFiNGE2MDIwYTk1NGZlM2E2OTU2ZmNlZjJmYjZkMTAiLCJwIjoiaiJ9)

I made it so the pagination dots can be orientated vertically. However, horizontal dots remain the default. I also added supporting documentation to [the docs](https://wpds-ui-kit-git-stry-301-vert-dots.preview.now.washingtonpost.com/components/pagination-dots).

https://wpds-ui-kit-git-stry-301-vert-dots.preview.now.washingtonpost.com/playroom?edit=1&code=KYDwDg9gTgLgBAE2AMwIYFcA29noHYDGMAlhHnAKIioC2YmwAFAJRwDeAUHHFMDOlHKMu3OAB4AQhBAjRcAgGcFAXjac5chMQX1UATwBccAETIGIYwBpZGs6AAixXkVJ4jxghEzoaeKze4AX0CbAD4A8SkZDVFFFTUIzW1dQxM7C2sYuQBzVDAjGAALYBpgADodVAJgAG0ARgAGBoBdTKzuSAViEjJ3XkxUEgA3YH924Ijw9rEABVRs4jxB13sIGAU4Wgh8GFUANkC4RaQQVTrDgYAjYExlYzB5xeWyRDWFYzgAeimNMU%2Bon5ySTSRLyJSqdTtRDJAapUzmMZQ3L5OBFErlSrVepNVqg7g0VBQBZ4CSYCAEADWAGUYISYAViqUKg8sQAmHFtdqdbquPo3ZYjRExCYxQG-ObE554VbrPGbGjbPC7NgHOXHUBnEJQuBXG53B6SnrkBBvYxy6DEYBKqV3EawYgEVCYM3tb4RP4Amwe6Q-ZgAbg4gSAA&isGuide=none

Screenshot 2024-05-07 at 3 44 29 PM

Play with the differing props in the dots to stress test the component.

[STRY-301]: https://arcpublishing.atlassian.net/browse/STRY-301?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: build.washingtonpost.com/docs/components/pagination-dots.mdx [hunk 1]
```diff
@@ -137,6 +137,49 @@ export default function Example() {
 }
 ```
 
+### Orientation
+
+If the dots are oriented left -> right or top -> bottom. It defaults to a horizontal orientation.
```
Comment (tamstephanie) [inline build.washingtonpost.com/docs/components/pagination-dots.mdx:142]: ⛏️ For consistency with other docs, could `left -> right` be written out fully, like `left to right`? (Same for `top -> bottom`)

File: packages/kit/src/pagination-dots/PaginationDots.tsx [hunk 2]
```diff
@@ -25,6 +25,8 @@ interface PaginationDotsProps extends React.ComponentPropsWithRef<"div"> {
   /** Specifies the type of element represented by the dots (e.g., "Page") */
   unitName?: string;
   css?: WPDS.CSS;
+  /** If the dots are oriented left -> right or top -> bottom */
+  orientation?: string;
```
Comment (tamstephanie) [inline packages/kit/src/pagination-dots/PaginationDots.tsx:29]: 💭 Would it be more valuable to set the type to only accept "horizontal" or "vertical"?

**Review by tamstephanie — APPROVED** [review]
Just two tiny suggestions, but otherwise, looks awesome!

**Review by tamstephanie — APPROVED** [review]

File: packages/kit/src/pagination-dots/PaginationDots.tsx [hunk 3]
```diff
@@ -134,7 +169,7 @@ function configureDots(nPages: number, activeIndex: number, amount: number) {
     // default: active dot has scale = 1, each dot previous/next's scale reduces by 25%
     let scale = 1 - stepsFromActive / 4;
     if (amount <= 5 && stepsFromActive > 1) {
-      // if there are 5 or less dots, make all dots at least 4px
+      // if there are 5 or fewer dots, make all dots at least 4px
```
Comment (lukaskeelwapo) [inline packages/kit/src/pagination-dots/PaginationDots.tsx:172]: 💯

**Review by lukaskeelwapo — APPROVED** [review]

**Review by artmsilva — APPROVED** [review]

**Pull request merged**